src/Elastic.Markdown/IO/MarkdownFile.cs

```diff
@@ -224,9 +230,9 @@ protected void ReadDocumentInstructions(MarkdownDocument document)
 
 		var toc = GetAnchors(_set, MarkdownParser, YamlFrontMatter, document, subs, out var anchors);
 
-		_tableOfContent.Clear();
+		_pageTableOfContent.Clear();
```

Comment:
While reading some navigation UX patterns I encountered the the term "On Page Table of Contents", which I think is very clear.

Reply:
I think it removes a lot of ambiguity, since in the HTML it also says "On this page"